### PR TITLE
Palindrome Number Without using extra space

### PR DESCRIPTION
**Pull request by anshmishra010**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

To check whether  a number is palindrome or not .Without using extra space .

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Palindrome number check

### CODE REVIEW

File: C-Plus-Plus/cp/Palindrome_number.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        divi*=10;
+    }
+    while(n!=0)
+    {
+        int l= n/divi;
+        int t = n%10;
+        if(l!=t){
+                return false;
+        }
+        n=(n%divi)/10;  //reducing number to compare
```
Comment (bhav09) [inline C-Plus-Plus/cp/Palindrome_number.cpp]: Comments should be above the lines of code not alongside.

File: C-Plus-Plus/cp/Palindrome_number.cpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/*
+Description :
```
Comment (bhav09) [inline C-Plus-Plus/cp/Palindrome_number.cpp:2]: Description can be formated well.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

File: C-Plus-Plus/cp/Palindrome_number.cpp [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/*
+Description :  
+            Palindrome Number : Given an integer x, return true if x is palindrome integer.
+                                An integer is a palindrome when it reads the same backward as forward.
+								For example, 121 is palindrome while 123 is not.
+                                This code is implemented without using extra space.
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Palindrome_number.cpp]: Format this and left-align the same

File: C-Plus-Plus/cp/Palindrome_number.cpp [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        if (l != t)
+        {
+            return false;
+        }
+        //reducing number to compare
+        n = (n % divi) / 10;
+        divi = divi / 100;
+    }
+    return true;
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Palindrome_number.cpp]: Use meaningful variable names

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**